assets/embauche.html

```diff
@@ -107,176 +107,224 @@
+		      <span class="value"><span data-source="agirc_employeur"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">ARRCO</span>
+		      <span class="value"><span data-source="arrco_employeur"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Assurance chômage</span>
+		      <span class="value"><span data-source="chomage_employeur"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">CET</span>
+		      <span class="value"><span data-source="cotisation_exceptionnelle_temporaire_employeur"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Allocations familiales</span>
+		      <span class="value"><span data-source="famille"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Vieillesse (déplafonnée)</span>
+		      <span class="value"><span data-source="vieillesse_deplafonnee_employeur"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Vieillesse (plafonnée)</span>
+		      <span class="value"><span data-source="vieillesse_plafonnee_employeur"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Invalidité</span>
+		      <span class="value"><span data-source="allocations_temporaires_invalidite"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Maladie</span>
+		      <span class="value"><span data-source="mmid_employeur"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Main d'œuvre</span>
+		      <span class="value"><span data-source="cotisations_employeur_main_d_oeuvre"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Contribution solidarité autonomie</span>
+		      <span class="value"><span data-source="contribution_solidarite_autonomie"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Formation professionnelle</span>
+		      <span class="value"><span data-source="formation_professionnelle"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Participation à l'effort de construction</span>
+		      <span class="value"><span data-source="participation_effort_construction"></span>&nbsp;€</span>
+		    </li>
+		    <li>
+		      <span class="title">Taxe d'apprentissage</span>
+		      <span class="value"><span data-source="taxe_apprentissage"></span>&nbsp;€</span>
+		    </li>
+				<li class="imprecise">
+		      <span class="title">Contribution supplémentaire à l'apprentissage</span>
+		      <span class="value"><span data-source="contribution_developpement_apprentissage"></span>&nbsp;€</span>
+					<div class="explanation">
+						<div>Calcul imprécis. Cette contribution décroît en fonction du pourcentage de l'effectif en apprentissage.</div>
+						<div class="estimate">
+							<span>
+								Taux possible :
+							</span>
+							<span class="interval">
+								<span class="from"> 0 %</span><span class="to"> 0,6 %</span>
```

Comment:
Keep the separating spaces between the `<span>` nodes, not inside. They will be mangled otherwise.